### PR TITLE
man: rework timer docs to use a table for monotonic timers

### PR DESCRIPTION
**Pull request by poettering**

### CODE REVIEW

File: man/systemd.timer.xml [hunk 1]
```diff
@@ -105,16 +105,42 @@
...
+                <entry><varname>OnActiveSec=</varname></entry>
+                <entry>Defines a timer relative to the moment the timer unit itself is activated.</entry>
+              </row>
+              <row>
+                <entry><varname>OnBootSec=</varname></entry>
+                <entry>Defines a timer relative to when the machine was booted up.</entry>
+              </row>
+              <row>
+                <entry><varname>OnStartupSec=</varname></entry>
+                <entry>Defines a timer relative to when the service manager was first started. For system timer units this is very similar to <varname>OnBootSec=</varname> as the system service manager is generally started very early at boot. It's primarily useful when configured in units running in the per-user service manager, as the user service manager is generally started on first login only, not already during boot.</entry>
```
Comment (boucman) [inline man/systemd.timer.xml:131]: howabout containers ? OnStartupSec is the container startup while OnBootSec is the host startup ? right ?

Comment (poettering) [reply man/systemd.timer.xml:131]: There's special code that tweaks things so that for containers OnBootSec= actually means container boot, not host boot.

Comment (boucman) [reply man/systemd.timer.xml:131]: maybe mention that ?

Comment (keszybz) [reply man/systemd.timer.xml:131]: I made a patch, but it seems I didn't push it in any PR.

Comment (boucman) [reply man/systemd.timer.xml:131]: thx

**Pull request merged**